lib/modules/dosomething/dosomething_reportback/includes/Reportback.php

```diff
@@ -162,8 +162,11 @@ private function build($data, $full = false) {
 
     if ($full) {
       $northstar_user = dosomething_northstar_get_northstar_user($data->uid);
-      $northstar_user = json_decode($northstar_user, true);
-      $northstar_user = (object) $northstar_user['data'][0];
+
+      if (!empty($northstar_user)) {
+        $northstar_user = json_decode($northstar_user, true);
+        $northstar_user = (object) $northstar_user['data'][0];
+      }
 
       $this->user = [
```

Comment:
Actually, scratch my suggestion above... it would make the returned data inconsistent and remove properties if `null` so wouldn't be as reliable... 

instead, maybe just wrap the calls when assigning the properties on the user array to use the `dosomething_helpers_isset()`:

```
...
  'first_name' => dosomething_helpers_isset($northstar_user->first_name),
...